Diagnostic/rsyslog5/omazurelinuxmds.conf

```diff
@@ -25,4 +25,4 @@ $ActionQueueSaveOnShutdown on
 $ActionQueueFileName lad_mdsd_queue
 $ActionQueueDiscardSeverity 8
 local6.* :omazuremds:;fmtfile
-& ~
+& stop
```

Comment:
@mabicca : Could you please double-check if this is supported on rsyslog v.5? If not, we shouldn't change this here.